data/json/effects.json

```diff
@@ -676,6 +676,60 @@
             "sleep_chance": [101]
         }
     },
+    {
+        "type": "effect_type",
+        "id": "bandaged",
+        "name": ["Bandaged"],
+        "desc": ["Your wounds are bandaged."],
+        "main_parts_only": true,
+        "rating": "good",
+        "int_dur_factor": 3600,
+        "max_intensity": 16,
+        "//": "How many intensity levels may apply effects",
```

Comment:
Don't comment known effects. Comments should document things which aren't in docs.
For example, `max_duration` - seems that it's equal to 4 days.